frontend/mobilityai_react/App/Screens/Navigator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/Entypo';
+import Menu, {
+  MenuOptions,
+  MenuOption,
+  MenuTrigger,
+} from 'react-native-popup-menu';
+
+const MenuIcon = () => <Icon 
+    name='dots-three-vertical' 
+    size={25} 
+    color='#000' 
+    style={{ marginRight: 10 }}
+  />;
+
+const optionsStyles = {
```

Comment:
You should follow the same format for styles that we use in the other components. Namely:

```js
const styles = StyleSheet.create({
...
});
```